Sources/Commands/SwiftPackageTool.swift

```diff
@@ -183,7 +183,24 @@ public struct SwiftPackageTool: SwiftTool {
                     return try get(manifest, manifestParser: parseManifest)
                 }
             }
-        
+
+            func findDirectoryReferences(path: String) throws -> [String] {
+                let rootDirectories = walk(path, recursively: false)
+                let rootDirectoriesToConsider = rootDirectories.filter {
+                    if $0.hasSuffix(".xcodeproj") { return false }
+                    if $0.hasSuffix(".playground") { return false }
+                    if $0.basename.hasPrefix(".") { return false }
+                    return $0.isDirectory
+                }
+
+                return rootDirectoriesToConsider.filter {
+                    let directoriesWithSwift = walk($0).filter {
+                        $0.hasSuffix(".swift")
```

Comment:
@Ruenzuo, do you plan to update this PR with changes based on these comments (moving it to Xcodeproj etc)?  I think it's a good change, but it would be good to address this comment about where the code should live.